### PR TITLE
Improve the API docs, module MD, and package MD

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$title

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: jballerina.java.arrays-ballerina/Package.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 ## Package Overview
+The Java arrays library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
```
Comment (daneshk) [inline jballerina.java.arrays-ballerina/Package.md:2]:
```suggestion

The Java arrays library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
```

Comment (daneshk) [reply jballerina.java.arrays-ballerina/Package.md:2]: shall we add new line in between

**Review by daneshk — APPROVED** [review]
LGTM

File: jballerina.java.arrays-ballerina/Module.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Module Overview
```
Comment (daneshk) [inline jballerina.java.arrays-ballerina/Module.md:1]:
```suggestion
## Overview
```

File: jballerina.java.arrays-ballerina/Package.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+This package provides a set of APIs to do Java array manipulations such as create new Java array instances, get elements from arrays, set elements.
+
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
+- Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
+- Post all technical questions on Stack Overflow with the #ballerina tag.
```
Comment (MadhukaHarith92) [inline jballerina.java.arrays-ballerina/Package.md:15]: Let's add the link here.

Comment (BuddhiWathsala) [reply jballerina.java.arrays-ballerina/Package.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/129/commits/a37d651dc60f2b308a187e568a7d4c4a981550c4

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

File: jballerina.java.arrays-ballerina/Package.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 ## Package Overview
 
-This package provides APIs to create new Java array instances, get elements from arrays, set elements, etc. 
+The Java arrays library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
+</a> language.
 
-For information on the operations, which you can perform with the jballerina.java.arrays package, see the below **Functions**. 
\ No newline at end of file
+This package provides a set of APIs to do Java array manipulations such as create new Java array instances, get elements from arrays, set elements.
```
Comment (praneesha) [inline jballerina.java.arrays-ballerina/Package.md:6]:
```suggestion
This package provides a set of APIs to do Java array manipulations such as create new Java array instances, get elements from arrays, set elements etc.
```

File: jballerina.java.arrays-ballerina/Package.md [hunk 5]
```diff
@@ -1,5 +1,16 @@
...
+
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```
Comment (praneesha) [inline jballerina.java.arrays-ballerina/Package.md:10]:
```suggestion
To report bugs, request new features, start new discussions, view project boards, etc., go to the Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**